habana/tensorflow/training/docker/2.9/py3/synapseai1.6.0/Dockerfile.hpu

```diff
@@ -0,0 +1,120 @@
+FROM vault.habana.ai/gaudi-docker/1.6.0/ubuntu20.04/habanalabs/tensorflow-installer-tf-cpu-2.9.1:latest
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# prevent stopping by user interaction
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
+# Set environment variables for MKL
+# For more about MKL with TensorFlow see:
+# https://www.tensorflow.org/performance/performance_guide#tensorflow_with_intel%C2%AE_mkl_dnn
+
+ENV KMP_AFFINITY=granularity=fine,compact,1,0
+ENV KMP_BLOCKTIME=1
+ENV KMP_SETTINGS=0
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV TF_VERSION=2.9
+
+ARG PYTHON=python3.8
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3.8
+
+ARG TF_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/habana/1.6.0/tensorflow_cpu-2.9.1-cp38-cp38-manylinux2014_x86_64.manylinux_2_17_x86_64.whl
+
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    build-essential \
+    openssh-client \
+    openssh-server \
+    ca-certificates \
+    curl \
+    jq \
+    emacs \
+    git \
+    libtemplate-perl \
+    libssl1.1 \
+    openssl \
+    unzip \
+    wget \
+    vim \
+    zlib1g-dev \
+    # Install dependent library for OpenCV
+    libgtk2.0-dev \
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get clean
+
+# SSH login fix. Otherwise user is kicked off after login
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
```

Comment:
Just a minor observation and not a concern for this PR. But just curious... Is this SSH login fix not required/needed for the PyTorch image? I see this directive only on the Tensorflow image and not on the other one.

Reply:
the change is not from us, I checked the code, it looks like added by Tejas Chumbalkar #1478, I don't have access to the commit.

Reply:
got it. thanks

Reply:
this file was sourced into the public repo by me and hence my name on the commit. In any case, it seems to be there from the start v0.15.4, lets keep it